python/reference-apps/python-gen-async/main.py

```diff
@@ -0,0 +1,7 @@
+async def save_entry_to_db(id: str) -> None:
```

Comment:
Maybe a test which run this sequence? Or you think it's not necessary?

Reply:
I can write one simple test I think